### ISSUE TITLE
[HybridApp] Clicking "Try New Expensify" multiple times will crash the app 💥 

### ISSUE THREAD

**Issue by AndrewGable** [Expensify/App#33467]
## Problem
If you tap the `Try New Expensify` row quickly in HybridApp, you can crash the app as it tries to display the new app more than once.

```
Fatal Exception: NSInvalidArgumentException
Application tried to present modally a view controller <UIViewController: 0x1057c77a0> that is already being presented by <ExpensifyRootViewController: 0x10570cc30>.
```

Stack trace: [com.expensify.expensifylite_issue_b9966109b55978d214bb1c0403b3dddd_crash_session_1bbf6a9e09614a36bfb5df48c90dc32a_DNE_0_v2_stacktrace.txt](https://github.com/Expensify/App/files/13746539/com.expensify.expensifylite_issue_b9966109b55978d214bb1c0403b3dddd_crash_session_1bbf6a9e09614a36bfb5df48c90dc32a_DNE_0_v2_stacktrace.txt)

## Solution
Only allow tap once, or debounce the tap so it can't display the view controller twice.

cc @staszekscp 

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0100d32dd2a521c2f3</li>
        <li>Upwork Job ID: 1737959299828318208</li>
        <li>Last Price Increase: 2023-12-21</li>
    </ul>
</details>

**Comment by AndrewGable** [comment]
@staszekscp is looking into this one

**Comment by AndrewGable** [comment]
This is fixed on the latest TestFlight `8.5.39.12`
